WordPress/WordPressTest/NSStringHelpersTest.m

```diff
@@ -116,4 +116,23 @@ - (void)testIsWordPressComPathWithInvalidProtocols
     }
 }
 
+- (void)testWordCount
+{
+    NSString *testEmptyPhrase = @"";
+    XCTAssert([testEmptyPhrase wordCount] == 0, @"Word count should be zero on a empty string");
+    
+    NSString *testPhraseEnglish = @"The lazy fox jumped over the fence.";
```

Comment:
Awesome! I'm being super picky but could you split each of these groupings into separate tests? `testWordCountEmpty`, `testWordCountEnglish`, etc. I'm a bit zealous sometimes about unit tests :grimacing:.